docs/canary/examples/daisyui.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Install daisyUI for Deno Fresh
+desc: How to install Tailwind CSS and daisyUI in a Deno Fresh project
+---
+
+This document records the complete process of integrating and using daisyUI in a
+Fresh v2.0 alpha project. The Fresh framework now supports the new Tailwind CSS
+plugin and Deno loader, with "nodeModulesDir": "auto" set by default, making the
+integration of daisyUI simpler and more efficient.
```

Comment:
- Remove mention of Fresh version number, the canary docs are all about v2 alpha
- Remove mention of Deno loader or "nodeModulesDir: "auto"`. Historical background information gets outdated quite quickly. That knowledge is not necessary for users to know when using daisyUI. Let's skip straight to the good parts.
- Explain in 1 or 2 sentences what daisyUI is and why it matters.